daemon/health.go

```diff
@@ -156,6 +157,7 @@ func monitor(d *Daemon, c *container.Container, stop chan struct{}, probe probe)
 			ctx, cancelProbe := context.WithTimeout(context.Background(), probeTimeout)
 			results := make(chan *types.HealthcheckResult)
 			go func() {
+				healthCheckCounter.Inc()
```

Comment:
Sounds good.  Do you want to try to implement this and see how it looks?  I'm not sure the best way to structure that but if you have an idea what you are looking for then let me know.